crates/ruma-client-api/src/profile/get_profile.rs

```diff
@@ -56,6 +64,11 @@ pub mod v3 {
         #[cfg(feature = "unstable-msc2448")]
         #[serde(rename = "xyz.amorgan.blurhash", skip_serializing_if = "Option::is_none")]
         pub blurhash: Option<String>,
+
+        /// The other fields of the user's profile.
+        #[cfg(feature = "unstable-msc4133")]
+        #[serde(flatten)]
+        pub others: BTreeMap<AnyProfileField, JsonValue>,
```

Comment:
When this lands in the spec, this could be the entire body of the response. It will be a breaking change sadly…

Another option is to make the breaking change now.